tests/testthat/test-subtotals-and-headings.R

```diff
@@ -184,4 +234,47 @@ with_test_authentication({
 
         expect_equivalent(v4_with_trans, v4_ary)
     })
+
+    test_that("Can modify subtotals in place", {
+        # assert known shape
+        expect_equal(names(subtotals(ds$v4)), c("This is a subtitle", "B alone",
+                                                "C alone", "B+C"))
+        expect_equal(anchors(subtotals(ds$v4)), c("top", 1, 2, "bottom"))
+        expect_equal(arguments(subtotals(ds$v4)[[1]]), NA)
+        expect_equal(arguments(subtotals(ds$v4)[[2]]), 1)
+        expect_equal(arguments(subtotals(ds$v4)[[3]]), 2)
+        expect_equal(arguments(subtotals(ds$v4)[[4]]), c(1,2))
+
+        # changing names
+        expect_silent(name(subtotals(ds$v4)[[1]]) <- "The new subtitle")
```

Comment:
why all the `expect_silent`? seems to mask (from my readability) the operations you're doing that are changing state, for little value.

Reply:
Old habits — I've taken them out.